### PR TITLE
Welcome

### PR DESCRIPTION
**Pull request by Stearnzy**
Create welcome page view and test.
One test skipped for User logging in happy path, will add next
Sad paths still need to be tested

Sessions controller created with New, Create, and Destroy, with paths (though unused at this point)

Current user added to Application Controller

### CODE REVIEW

File: app/controllers/application_controller.rb [hunk 1]
```diff
@@ -1,3 +1,10 @@
 class ApplicationController < ActionController::Base
   protect_from_forgery with: :exception
+
+  def current_user
+    begin
+      @current_user ||= User.find(session[:user_id]) if session[:user_id]
+    rescue
```
Comment (stillsheryl) [inline app/controllers/application_controller.rb:7]: Is this necessary? Do we need some kind of error message to redirect?

Comment (Eternal-Flame085) [reply app/controllers/application_controller.rb:7]: this is necessary to get the current user if one is not already logged in

**Review by Eternal-Flame085 — APPROVED** [review]

File: app/views/welcome/index.html.erb [hunk 2]
```diff
@@ -0,0 +1,11 @@
+<h1>Welcome to Viewing Party!</h1>
+
+<h3 id="description">Finally, a place where you can enjoy all your favorite movies with all your best friends!</h3>
+
+<%= form_with url: login_path, local: true do |f| %>
+  <%= f.text_field :email, placeholder: 'Enter Email' %>
```
Comment (stillsheryl) [inline app/views/welcome/index.html.erb:6]: Nice addition with the placeholders!

**Review by stillsheryl — APPROVED** [review]
Looks great!

File: app/controllers/application_controller.rb (hunk 1 above)
Comment (stillsheryl) [reply app/controllers/application_controller.rb:7]: Awesome, thanks!

**Pull request merged**